tests/common/ixia/ixia_fixtures.py

```diff
@@ -193,11 +194,31 @@ def ixia_api(ixia_api_serv_ip,
     if api_session:
         api_session.assistant.Session.remove()
 
+
+@pytest.fixture(scope='module')
+def snappi_api(ixia_api_serv_ip,
```

Comment:
don't use `ixia`

Reply:
as discussed, we need a meeting with Wei